fakesystem.spec

```diff
@@ -107,6 +107,10 @@ Provides: perl(sort)
 # ROCM
 #####
 Provides: libamdhip64.so.5(hip_5.3)(64bit)
+Provides: libdrm.so.2()(64bit)
+Provides: libdrm_amdgpu.so.1()(64bit)
+Provides: perl(File::Which)
+Provides: perl(URI::Encode)
```

Comment:
@fwyzard , for now I would suggest to add these to the package which needs these. Once we have a working PR then we can move these requires in to fakesystem and update the revision

Reply:
OK, I will do it like that for the local tests, and then push them here once they work locally.

Reply:
How can I test the `fakesystem` changes locally ?

Reply:
ah, ok !